README.md

```diff
@@ -1,40 +1,41 @@
-<img src="http://moveit.ros.org/assets/images/moveit2_logo_black.png" alt="MoveIt Logo" width="200"/>
+<img src="https://moveit.ros.org/assets/logo/moveit_logo-black.png" alt="MoveIt Logo" width="200"/>
 
-The MoveIt Motion Planning Framework
+**The MoveIt Motion Planning Framework**
 
-Currently we support ROS Kinetic and Melodic for ROS 1.
+*Easy-to-use open source robotics manipulation platform for developing commercial applications, prototyping designs, and benchmarking algorithms.*
 
 - [Overview of MoveIt](http://moveit.ros.org)
 - [Installation Instructions](http://moveit.ros.org/install/)
 - [Documentation](http://moveit.ros.org/documentation/)
-- [Get Involved](http://moveit.ros.org/documentation/contributing/)
+- [Get Involved](https://moveit.ros.org/about/get_involved/)
 
 ## Branches Policy
 
-We develop all new 1.0 features on ``master``. The ``*-devel`` branches correspond to
-released and stable versions of MoveIt for specific distributions of ROS.
-Bug fixes occationally get backported to these released versions of MoveIt.
-The next version of MoveIt 1.0 will be branched to ``noetic-devel`` around May 2020.
+- We develop latest features on ``master``.
+- The ``*-devel`` branches correspond to released and stable versions of MoveIt for specific distributions of ROS.
+- Bug fixes occationally get backported to these released versions of MoveIt.
+- The next version of MoveIt 1.0 will be branched to ``noetic-devel`` around June 2020.
```

Comment:
```suggestion
```

Reply:
That's going to be outdated within days.

Reply:
I don't see this as an issue... once it is outdated it will be obvious because of the timestamp.

What if someone wants to know about Noetic? This will help them. We will probably forget to add new information here for a while anyway, so this is way better than nothing.